### DISCUSSION TITLE
Add media to meeting - any hooks available for use?

### DISCUSSION THREAD

**Discussion by koen-dieren** [code4recovery/12-step-meeting-list#1287] [General]
Hi! I'm using the TSML UI interface, and want to add the possibility to add images to a meeting page with a custom field. Are there any hooks that I can use to display it under the map?

**Comment by anchovie91471** [comment]
This isn't possible with TSML UI. 

It can be done using a theme override with the Legacy interface (single-meetings.php) and a plugin that adds custom fields to post types or native Wordpress hooks that override post types.

**Reply by koen-dieren** [reply]
Thank you for the feedback.

**Comment by joshreisner** [comment]
out of curiosity, what types of images did you want to add to the meeting listing?

**Reply by koen-dieren** [reply]
Images of the location.
